portal/migrations/versions/a1de3ab4d050_.py

```diff
@@ -0,0 +1,118 @@
+"""Remove duplicate QNR.identifiers
+
+Revision ID: a1de3ab4d050
+Revises: 55469bdd181f
+Create Date: 2019-02-11 16:48:55.332527
+
+"""
+import json
+from sqlalchemy import func
+from portal.database import db
+from portal.dict_tools import dict_compare
+from portal.models.audit import Audit
+from portal.models.identifier import Identifier
+from portal.models.questionnaire_response import QuestionnaireResponse
+from portal.models.role import Role, ROLE
+from portal.models.user import User, UserRoles
+
+# revision identifiers, used by Alembic.
+revision = 'a1de3ab4d050'
+down_revision = '55469bdd181f'
+
+
+def admin_id():
+    """Look up an admin ID"""
+    admin = User.query.filter_by(email='bob25mary@gmail.com').first()
```

Comment:
Can we use the `__system__` email for migrations?

Reply:
oh!  forgot about that option.  thanks.